src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.h

```diff
@@ -67,6 +69,12 @@ class CARMAStreetsPlugin: public PluginClientClockAware {
 	 * @param routeableMsg 
 	 */
 	void HandleMapMessage(MapDataMessage &msg, routeable_message &routeableMsg);
+	/**
+	 * @brief Subscribe to SDSM broadcast.
```

Comment:
Lets have a better description here. This method handles incoming SDSMs received by RSU and forwards them to CARMA Streets as JSON string messages.